VOC2007/ImageSets/Main/aeroplane_train.txt

```diff
@@ -0,0 +1,2501 @@
+000012 -1
```

Comment:
Don't push any external files to the repo. please remove all pascal files from the repo.